tools/building.py

```diff
@@ -483,9 +483,9 @@ def llvm_backend_args():
   args = ['-combiner-global-alias-analysis=false']
 
   # asm.js-style exception handling
-  if Settings.DISABLE_EXCEPTION_CATCHING != 1:
+  if not Settings.DISABLE_EXCEPTION_CATCHING:
     args += ['-enable-emscripten-cxx-exceptions']
-  if Settings.DISABLE_EXCEPTION_CATCHING == 2:
+  if Settings.EXCEPTION_CATCHING_ALLOWED:
     allowed = ','.join(Settings.EXCEPTION_CATCHING_ALLOWED or ['__fake'])
```

Comment:
Do we still need `__fake`? Not sure why it was there in the first place, but now we don't enter this `if` if `EXCEPTION_CATCHING_ALLOWED`is empty anyway.

Reply:
Hmm indeed, I think this can just be removed.